package.json

```diff
@@ -25,7 +25,7 @@
     "react-dom": "^16.7.0",
     "react-github-corner": "^2.3.0",
     "react-router-dom": "^4.3.1",
-    "react-scripts": "^2.1.3",
+    "react-scripts": "^5.0.1",
```

Comment:
In an earlier PR, I mentioned that this is not that simple.
The same applies here.